packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -4414,7 +4414,7 @@ void main() {
         const TextSelection(
           baseOffset: 20,
           extentOffset: 20,
-          affinity: TextAffinity.downstream,
+          affinity: TextAffinity.upstream,
```

Comment:
Previously, we were usually resetting the affinity when manipulating the selection.  I updated everything to preserve the affinity, or explicitly set it when it makes sense (like moving the selection to the start should be upstream).  I updated these tests to match.